test/bridge.test.ts

```diff
@@ -443,19 +435,20 @@ describe("Bridge", async function () {
     expect(await l2StaticAUsdc.call('get_last_update', {})).to.deep.equal({ block_number: {high: 0n, low: BigInt(blockNumberUsdc) }});
 
     // check balance of L2 tokens
-    expect(await l2StaticADai.call('balanceOf', { account: BigInt(l2user.starknetContract.address) })).to.deep.equal({ balance: { high: 0n, low:  28n } });
-    expect(await l2StaticAUsdc.call('balanceOf', { account: BigInt(l2user.starknetContract.address) })).to.deep.equal({ balance: { high: 0n, low:  37n } });
+    l2staticADaiBalance = await l2StaticADai.call('balanceOf', { account: BigInt(l2user.starknetContract.address) });
+    l2staticAUsdcBalance = await l2StaticAUsdc.call('balanceOf', { account: BigInt(l2user.starknetContract.address) });
+    expect(BigNumber.from(27n * DAI_UNIT)).to.be.lte(l2staticADaiBalance["balance"]["low"]);
+    expect(BigNumber.from(37n * USDC_UNIT)).to.be.lte(l2staticAUsdcBalance["balance"]["low"]);
   })
 
-
   it('Withdraws Dai and Usdc', async () => {
     // approve L2 bridge with given amount
-    await l2user.invoke(l2StaticADai, 'approve', { spender: BigInt(l2StaticADai.address), amount: { high: 0n, low:  28n } });
-    await l2user.invoke(l2StaticAUsdc, 'approve', { spender: BigInt(l2StaticAUsdc.address), amount: { high: 0n, low:  37n } });
+    await l2user.invoke(l2StaticADai, 'approve', { spender: BigInt(l2StaticADai.address), amount: { high: 0n, low: 30n * DAI_UNIT } });
```

Comment:
I don't think we need to approve anything when withdrawing  l2 staticATokens, since they are burnt at the moment of the withdrawal by the l2 bridge (which is the admin of all staticATokens).

Reply:
you're right, this part of the code does not make sense. we're approving the token - and not the bridge as the comment above states - using its own address as spender - while we should set a user address. removing it